appconfig/settings/default.q

```diff
@@ -3,18 +3,17 @@ system"c 23 2000"
 .usage.logtodisk:0b; / disable usage logging as we cannot write to disk in finspace
 
 .servers.FINSPACEDISC:1b;
-.servers.FINSPACECLUSTERSFILE:hsym `$getenv[`KDBAPPCONFIG],"/clusters.csv"
 
-.servers.CLUSTERS:("SSSS**"; enlist ",") 0: .servers.FINSPACECLUSTERSFILE;
-.servers.SOCKETTYPE:{x!count[x]#`finspace} exec distinct proctype from .servers.CLUSTERS;
+//-- .servers.procstab will be overwritten by trackservers.q on servers initiation
+.servers.SOCKETTYPE:{x!count[x]#`finspace} exec distinct proctype from .servers.procstab:("S*SS*BBHH*B**";enlist ",") 0: .proc.file;
 
 .finspace.enabled:1b;
 
 // hb subscriptions keeps the connections alive
 .hb.subenabled:1b;
 
-svrstoload:select from .servers.CLUSTERS where proctype = .proc.proctype
-$[count toload:first exec toload from svrstoload where procname = .proc.procname;
+svrstoload:select from .servers.procstab where proctype = .proc.proctype;
+$[count toload:first ?[svrstoload;enlist (=;`procname;(),.proc.procname);();`load];
```

Comment:
why functional form? isn't this just

```suggestion
$[count toload:first exec load from srvrstoload where procname=((),.proc.procname);
```

or maybe `enlist .proc.procname` & get rid of all those brackets? why does it need to be enlisted anyway? and actually, isn't this filtering already done by the line above?

Reply:
process.csv has the column ``` load ``` (whereas cluster.csv had the column ``` toload ```). When using the qsql, the kdb interpreter sometimes confuses the "load" column with the "load" keyword.

Reply:
The line above filters by ``` proctype ```. Subsequent queries filter that further down by ``` procname ```.

Reply:
Instead of using functional form, can we do something like

``(select from serverstoload where procname=.proc.procname)`load``

Think this is a little more readable